client/src/app/+login/login.component.html

```diff
@@ -8,73 +8,81 @@
     </div>
 
-    <div class="login-form-and-externals">
-
-      <form role="form" (ngSubmit)="login()" [formGroup]="form">
-        <div class="form-group">
-          <div>
-            <label i18n for="username">User</label>
-            <input
-              type="text" id="username" i18n-placeholder placeholder="Username or email address" required tabindex="1"
-              formControlName="username" class="form-control" [ngClass]="{ 'input-error': formErrors['username'] }" #usernameInput
-            >
-            <a i18n *ngIf="signupAllowed === true" routerLink="/signup" class="create-an-account">
-              or create an account
-            </a>
-          </div>
-
-          <div *ngIf="formErrors.username" class="form-error">
-            {{ formErrors.username }}
+    <div class="wrapper">
+      <div class="login-form-and-externals">
+
+        <form role="form" (ngSubmit)="login()" [formGroup]="form">
+          <div class="form-group">
+            <div>
+              <label i18n for="username">User</label>
+              <input
+                type="text" id="username" i18n-placeholder placeholder="Username or email address" required tabindex="1"
+                formControlName="username" class="form-control" [ngClass]="{ 'input-error': formErrors['username'] }" #usernameInput
+              >
+            </div>
+
+            <div *ngIf="formErrors.username" class="form-error">
+              {{ formErrors.username }}
+            </div>
           </div>
-        </div>
 
-        <div class="form-group">
-          <label i18n for="password">Password</label>
-          <div>
+          <div class="form-group">
+            <label i18n for="password">Password</label>
             <my-input-toggle-hidden formControlName="password" id="password"
                                     i18n-placeholder placeholder="Password"
                                     [ngClass]="{ 'input-error': formErrors['password'] }"
-                                    autocomplete="current-password"></my-input-toggle-hidden>
-            <a i18n-title class="forgot-password-button" (click)="openForgotPasswordModal()" title="Click here to reset your password">I forgot my password</a>
+                                    autocomplete="current-password" tabindex="2"></my-input-toggle-hidden>
+            <div *ngIf="formErrors.password" class="form-error">
+              {{ formErrors.password }}
+            </div>
           </div>
-          <div *ngIf="formErrors.password" class="form-error">
-            {{ formErrors.password }}
+
+          <input type="submit" i18n-value value="Login" [disabled]="!form.valid">
+          
+          <div class="additionnal-links">
+            <a i18n-title class="forgot-password-button" (click)="openForgotPasswordModal()" title="Click here to reset your password">I forgot my password</a>
+            <div *ngIf="signupAllowed" class="signup-link">
+              <span>·</span>
```

Comment:
What's this? :)

Reply:
It’s a visual separator, that could be implemented otherwise with a css `::before` rule :slightly_smiling_face:

Reply:
Ah, I see, I thought it was left there by mistake.